src/CsvHelper/CsvParser.cs

```diff
@@ -117,7 +116,7 @@ public virtual string[] Read()
 				if( configuration.DetectColumnCountChanges && row != null )
 				{
 					if( FieldCount > 0 && ( FieldCount != row.Length || 
-#if NET_2_0
+#if NET_2_0 && !PCL

```

Comment:
There are a few of these to include support for PCL, but using the .NET 2.0 profile - used for iOS only